protocols/enocean/README.md

```diff
@@ -81,7 +81,8 @@ Note
 ----
 Using the example app/bundle (org.eclipse.smarthome.protocols.enocean.sample.client) requires to 
 put the following lines in the enocean_config.txt configuration file of ``org.eclipse.smarthome.
-protocols.enocean.basedriver.impl```:
+protocols.enocean.basedriver.impl```:

```

Comment:
yes, looks so. Guess a typo in Kai's earlier commit
https://github.com/eclipse/smarthome/blame/ae20c3fa64baa95e50b551a25ee0ee37d78649b7/protocols/enocean/README.md#L83